__tests__/InputTest.js

```diff
@@ -0,0 +1,38 @@
+import ERROR_MESSAGE from "../src/constants/errorMessage.js";
+import { validateCarNameList, validateTryCount } from "../src/validation.js";
+
+describe("자동차 입력 테스트", () => {
+  test("자동차 이름 구분자가 쉼표인 경우", async () => {
+    // given
+    const inputs = "phobi, woni";
+
+    // when, then
+    expect(() => validateCarNameList(inputs)).toBeTruthy();
```

Comment:
이 부분에서는 꼭 `toBeTruthy`를 써야할까 개인적으로는 의문인 부분이에요

우선 첫째로 validationCarNameList를 검증에 실패하면 throw error를 던지고 성공할 경우 true를 던지는데 `truthy`한 값을 검증해야하는가 하는 의문이 있어요 throw error 때문에 `toBe(true)` 검증이 어렵다고 생각하신거라면 `not.toThrow()`로 검증하는게 더 타당하지 않을까 하는 생각이에요

두번쨰로 `expect(() => validateCarNameList(inputs)).toBeTruthy();` 라는 코드는 validateCarNameList라는 함수 호출을 래핑하고  있어서 `실제 함수의 반환값 검증`이 아니지 않나 하는 생각이 들어요 그래서 


```suggestion
 // 만약 기존처럼 toBeTruthy를 사용하실 예정이시면
   expect(validateCarNameList(inputs)).toBeTruthy();
   
   // 만약 not.throw로 테스트하려면
    expect(() => validateCarNameList(inputs)).not.toThrow();
 
```

이렇게 수정하는게 더 맞지 않을까 생각이 들어요! 아니면 둘 다 복합적으로 사용하여 성공/에러 발생시로 테스트해 보는것도 좋을 것 같아요! 아 not.toThrow는 함수 호출을 람다 함수로 감싸는 이유는 실행 여부를 검증하는 것이지 값 자체를 테스트하는 것이 아니기 때문이에요 (저는 그렇게 학습했습니다!)

Reply:
헉 디테일한 피드백 너무 너무 감사드려요!
해당 부분 로직을 작성할 때, 실행을 위한 부분에선 boolean return문이 없었는데 테스트를 하기 위해 추가했었답니다..
not.toThrow()의 올바른 활용법을 몰라서 생긴 일이었습니다. 😢

더불어 함수로 감싸는 것보다 실제 함수의 반환값 검증이 이루어지도로록 함수 호출을 래핑없이 사용하도록 해볼게요! 감사합니다 😄⭐